src/Centreon/Domain/HostConfiguration/Interfaces/HostGroup/HostGroupReadRepositoryInterface.php

```diff
@@ -52,6 +53,15 @@ public function findAll(): array;
      */
     public function findAllByContact(ContactInterface $contact): array;
 
+    /**
+     * Find all host groups linked to a host.
+     *
+     * @param Host $host
+     * @return HostGroup[]
+     * @throws \Throwable
+     */
+    public function findByHost(Host $host): array;
```

Comment:
method name seems not consistent with previous method `findAllByContact`
May need to be renamed `findAllByHost`